ballerina-command/src/main/resources/cli-help/ballerina-dist.help

```diff
@@ -1,14 +1,14 @@
 NAME
-       Manage Ballerina distributions
+       ballerina-dist - Manage Ballerina distributions
 
 SYNOPSIS
-       ballerina dist <command> <-h | --help>
-       ballerina dist <command> [<args>]
+       ballerina dist [-h | --help]
+                      <command> [<args>]
```

Comment:
`ballerina dist -h list` does not print the help text though, which is why we thought of introducing two.

Reply:
This is a standard format AFAIK. It is there even in our ballerina help output. You can try "git" and see the output. :)

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/20449 to track fixing this.